django/core/handlers/base.py

```diff
@@ -105,6 +105,15 @@ def get_exception_response(self, request, resolver, status_code, exception):
 
     def get_response(self, request):
         "Returns an HttpResponse object for the given HttpRequest"
+        def process_exception_by_middlewares(exception):
```

Comment:
Is there any benefit to making this an inner method as opposed to putting it on the class?